benchalerts/tests/unit_tests/test_pipeline_steps/test_github.py

```diff
@@ -19,6 +21,20 @@
 )
 
 
+class MockAlerter(Alerter):
+    def github_check_summary(
+        self, full_comparison: FullComparisonInfo, build_url: Optional[str]
+    ) -> str:
+        old_message = super().github_check_summary(full_comparison, build_url)
```

Comment:
"old" is perhaps not the best adjective here; "original" or "base" is more apt

Reply:
haha true. I'll fix.

>I wonder if templating for creating messages instead of all the += would make it easier? 

Yeah it would be easier, but I'm afraid it would be less flexible. Maybe after a few implementations we can reevaluate if it's possible? I suspect most implementations will just want to overwrite the intro sentence anyway, which one can do by overriding the `intro_sentence()` method.